.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+
+**Thank your for opening a PR!**
+
+Depending on what your PR does, here are a few things you might want to address in the description:
++ [ ] what are the (breaking) changes that this PR makes?
++ [ ] important background, or a details about the implementation
++ [ ] are the changes covered by tests? 
++ [ ] are example-notebooks affected in a relevant way?
```

Comment:
should also add whether doc-strings and examples are added if it's a new feature.

Reply:
good point - I re-formulated point 3 and 4 to cover them

Reply:
I think the point we should add is whether a potential new feature is properly documented or not.